include/deal.II/base/trilinos_utilities.h

```diff
@@ -180,6 +184,34 @@ namespace Utilities
     duplicate_map(const Epetra_BlockMap &map, const Epetra_Comm &comm);
   } // namespace Trilinos
 #endif
+
+
+#ifdef DEAL_II_TRILINOS_WITH_TPETRA
+  namespace Trilinos
+  {
+    /**
+     * Creates and returns a
+     * <a
+     * href="https://docs.trilinos.org/dev/packages/teuchos/doc/html/classTeuchos_1_1RCP.html">Teuchos::RCP</a>
+     * object for type T.
+     *
+     * @note In Trilinos 14.0.0, the function
+     * <a
+     * href="https://docs.trilinos.org/dev/packages/teuchos/doc/html/namespaceTeuchos.html#a280c0ab8c9ee8d0481114d4edf5a3393">Teuchos::make_rcp()</a>
+     * was introduce. If Trilinos version 14.0.0 or higher is detected,
+     * we use an alias for that function. Otherwise, we make our own version
+     * of make_rcp().
```

Comment:
```suggestion
     * was introduce.
```
I don't think we need to expose implementation details.